stories/form/form-date-input/form-date-input.stories.mdx

```diff
@@ -24,24 +26,107 @@ The field that allows a user to enter a date value.
 import { Form } from "@lifesg/react-design-system/form";
 ```
 
-> **Important note**: This component currently does not do date value validation
+<Heading3>Using a default date input</Heading3>
 
 <Canvas>
     <Story name="DateInput">
         <StoryContainer>
             <Container>
-                <Form.DateInput label="This is the date input" />
+                <Form.DateInput label="This is the default date input" />
+                <Form.DateInput
+                    label="This is the date input without button"
+                    withButton={false}
+                />
                 <Form.DateInput label="This is the disabled state" disabled />
-                <Form.DateInput label="This is the readonly state" readOnly />
+                <Form.DateInput label="This is the readOnly state" readOnly />
+            </Container>
+        </StoryContainer>
+    </Story>
+</Canvas>
+
+<Heading3>Using a range selection</Heading3>
+
+You can use `variant='range'` to specify to range selection.
+
+<Canvas>
+    <Story name="Ranged DateInput">
```

Comment:
```suggestion
    <Story name="Range input">
```

Reply:
Also for this story, can you add a state to it. Right now because of the lack of state, I cannot see the range display properly